app/controllers/hiring_staff/vacancies/publish_controller.rb

```diff
@@ -8,12 +8,20 @@ def create
       reset_session_vacancy!
       redirect_to school_job_summary_path(vacancy.id)
     else
+      skip_to_change_publish_on_date(vacancy.id) && return if vacancy.publish_on.past?
```

Comment:
When there are any errors when publishing we just want to display the generic error message and leave the user on the review page.

In the *How migh we session* we run last week, we did decided for some changes to be implemented so that more information is displayed to enable users to identify any old draft vacancy errors, but that work will be done as part of https://trello.com/c/x14tp4v4/487-display-detailed-error-messages-if-vacancy-publication-fails